doc/dynamic-data.md

```diff
@@ -24,6 +24,49 @@ This is the information available in each provider.
 | OpenStack-Metadata | ✓          | ✓              | ✓             |                |               |
 | Vagrant-Virtualbox | ✓          | ✓              |               |                |               |
 
+## Custom metadata providers
+
+As of ct 7.0(FIXME: what do we put here), ct also supports custom metadata providers. To use the `custom` platform, overwrite the coreos-metadata service unit to execute your own custom metadata fetcher. The custom metadata fetcher much write an enviroment file with all of the dynamic fields used to `/run/metadata/coreos`.
```

Comment:
imo we don't need to say what version this exists in, just release a new version with this quick-ish and if people ask we can show them the NEWS file.

Reply:
s/much/must/
s/enviroment/environment/
`dynamic fields used to /run/metadata/coreos`?